service/src/main/java/tds/assessment/repositories/impl/ItemQueryRepositoryImpl.java

```diff
@@ -63,7 +63,7 @@ public ItemQueryRepositoryImpl(final NamedParameterJdbcTemplate jdbcTemplate) {
                 "JOIN \n" +
                 "   itembank.tblitem item \n" +
                 "   ON item._key = adminItems._fk_item \n" +
-                "JOIN \n" +
+                "LEFT JOIN \n" +
```

Comment:
Adaptive items do not have forms and are not present in testformitem.  So whenever we tried to get items for an adaptive exam we were getting 0 items.